### PR TITLE
Vpc support

### PR DESCRIPTION
**Pull request by Slony**
Adds VPC support for using Cloud SQL instance without public IP address

### CODE REVIEW

**Review by instant-bqml — APPROVED** [review]
Thank you!

**Review by dulacp — COMMENTED** [review]
The test is failing because the click.invoke method by default catches exception.

You can first disable this behavior (my bad for not doing it previously):

```
result = runner.invoke(
    _custom_command, standalone_mode=False, catch_exceptions=False)
```

Then you will see the underlying exception, stage file is missing the flag, could be fixed with:

```
stage = shared.default_stage_context(shared.ProjectId('foo'))
stage.use_vpc = False
```

**Review by dulacp — APPROVED** [review]
LGTM

File: cli/utils/constants.py [hunk 1]
```diff
@@ -101,9 +101,7 @@
 
     # Network configuration
     network = "{ctx.network}"
```
Comment (dulacp) [inline cli/utils/constants.py:103]: Needs a `use_vpc = "{ctx.use_vpc}"`.

You will also need to change the `testdata/dummy_stage_v3.py` stage file accordingly.

**Review by dulacp — APPROVED** [review]
The tests are failing, because the `use_vpc` won't be saved in the final stage file.

**Pull request merged**